### DISCUSSION TITLE
reopt bad request

### DISCUSSION THREAD

**Discussion by xyl010203** [urbanopt/urbanopt-cli#487] [General]
At first, I had  problem of make request failed.Then I updated the certificate by running the certified-update file in the URBANopt-cli-0.13.0\gems\ruby\2.7.0\gems\certified-1.0.0\bin directory. Now it can run, but it reports an error with the code 400. I have set up proxies for both http and https on Git Bash to access web resources through the port of Clash. The error message is as follows.

"""Post-processing entire scenario with REopt
W, [2024-10-29T21:25:55.658066 #16120]  WARN -- : REopt has returned a '400' status code. Visit https://developer.nrel.gov/docs/errors/ for more status code information
E, [2024-10-29T21:25:55.661562 #16120] ERROR -- : MESSAGES: {"errors"=>{"core_urdb.jl_134"=>[" Error occurred HTTP.Exceptions.StatusError(500", " GET", " utility_ratesapi_key2qt5uihpKXdywTj3uMIhBewxY9K4eNjpRje1JUPLversion8formatjsondetailfullgetpage63d2f36655296095a0092a6e", " HTTP.Messages.ResponsennHTTP1.1 500 Internal Server ErrorrnDate Tue", " 29 Oct 2024 132543 GMTrnContentType texthtml charsetUTF8rnContentLength 0rnConnection keepalivernAccessControlAllowOrigin rnAge 131rnExpectCt maxage7776000", " reporturihttpsopenei.orgwikiOpenEIAboutrnStrictTransportSecurity maxage31536000preload", " maxage31536000 preloadrnVia ma4", " fs8.openei.org", " https1.1 apiumbrella (ApacheTrafficServer cMsSf )rnXApiUmbrellaRequestId cir1p2aube77vjkft940rnXCache MISS from fs8.openei.orgrnXContentTypeOptions nosniffrnXRatelimitLimit 1000rnXRatelimitRemaining 984rnXVcapRequestId 6709a4972c33417850dddad5dcf40347rnXXssProtection 1 modeblockrnXFrameOptions sameoriginrnrn)"]}, "has_stacktrace"=>false}
response.code: 400
message: Bad Request
"""
would anybody help me with my problems.thank u

**Comment by kflemin** [comment]
We have recently released newer versions of URBANopt which addresses many prior issues. If you are still experiencing issues related to this after upgrading, please let us know.
